internal/api/api.go

```diff
@@ -67,21 +72,28 @@ type idler struct {
 	openShiftClient client.OpenShiftClient
 	controller      openshift.Controller
 	tenantService   tenant.Service
+	statusService   userstatus.Service
 }
 
 type status struct {
 	IsIdle bool `json:"is_idle"`
 }
 
+type userStatus struct {
+	DisableUserList []string `json:"disable_users"`
+	EnableUserList  []string `json:"enable_users"`
```

Comment:
could you please explain why we need an enabled user list? doesn't absence of use in `disabled` list imply idler is enabled for user?

Reply:
If one's user is set to disable list we don't have a way to enable it again, in order to again enable the user we need to remove.

Reply:
@pradeepitm12 thanks, makes sense to me now. 
On a related note, shouldn't `enable`/`disable` do instead of `disabled_users` and `enable_users`?